index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="main-nav">
+            <div class="left-bar">
+                <p class="time">9:41</p>
+                <p class="name">Oscar Byiringiro</p>
+            </div>
+            <div class="right-bar">
+                <div class="top-icons">
+                    <img src="./images/Cellular Connection.png">
+                    <img src="./images/Wifi.png">
+                    <img src="./images/Battery.png">
+                                
+                            
+                </div>
+                <div class="bottom-icons">
+                    <img src="./images/Union.png">
+                        
+                </div>
+                
+                    
+            </div>
+        </nav>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <p>Hey There</p>
+                    <p>I'm Oscar</p>
+                </div>
```

Comment:
- [ ] Great job adding your introduction 👍 However, as suggested please make sure that you use semantic `<h1>` tag for this instead of using `<p>` tag. Also, It is important to use the `<h1>` tag only once per page, and only for the main heading of the page. This will help to ensure that the semantic meaning, accessibility, SEO and design benefits of the `<h1>` tag are maximized. For more clarity, please visit [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). I know you can do that 💪